doc/intro.rst

```diff
@@ -6,7 +6,7 @@ asyncio system module. It requires Python 3.5 or later, as it makes extensive
 uses of coroutines and type annotations. It is loosely inspired by the `Python
 2 bindings`_, but has a much narrower scope.
 
-.. _katcp: https://katcp-python.readthedocs.io/en/latest/_downloads/NRF-KAT7-6.0-IFCE-002-Rev5.pdf
+.. _katcp: https://katcp-python.readthedocs.io/en/latest/_downloads/361189acb383a294be20d6c10c257cb4/NRF-KAT7-6.0-IFCE-002-Rev5-1.pdf
```

Comment:
Hopefully this link persists after a new version of the docs is built.

Reply:
I hope so too... I don't know what the magic hex string represents, but it's not a md5sum, sha1sum or sha256sum of the file. If it breaks when you do the next release I'll give up and just put a copy into aiokatcp.